contracts/TracerPerpetualSwaps.sol

```diff
@@ -33,6 +33,8 @@ contract TracerPerpetualSwaps is ITracerPerpetualSwaps, Ownable {
     uint256 public override fees;
     address public override feeReceiver;
 
+    uint256 private constant MAX_PERCENT = 1e20; // Constant for 100% in WAD format, used to check valid parameters
```

Comment:
Slightly nicer way of writing this would be `100e18` (conveys that it's 100 in WAD)